pylintrc

```diff
@@ -416,7 +416,7 @@ max-spelling-suggestions=2
 max-args = 9
 
 # Maximum number of locals for function / method body
-max-locals = 19
+max-locals = 20
```

Comment:
maybe it would be better to locally ignore this for the one function that apparently now needs this high limit.